site/gatsby-site/src/components/forms/SubmitForm.js

```diff
@@ -86,6 +91,10 @@ const SubmitForm = () => {
       }
     }
 
+    if (queryParams['tags'].includes(RESPONSE_TAG) && queryParams['incident_id']) {
```

Comment:
Technically the same, but why use `queryParams['tags']` instead of `queryParams.tags`?